wallet_site/app/main.py

```
@@ -14,14 +15,18 @@
 from app.service import list_jobs_by_user
 from app.wallet_service import (
     import_wallets_from_json, get_user_wallets, export_wallets,
-    query_balances, delete_wallets
+    query_balances, delete_wallets, update_wallet_name
 )
 from app.models import Wallet
```

Ruff (0.12.2)

`app.models.Wallet` imported but unused

Remove unused import: `app.models.Wallet`

(F401)